src/runtime_src/xdp/profile/writer/aie_debug/aie_debug_writer.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <vector>
+
+#include "xdp/profile/writer/aie_debug/aie_debug_writer.h"
+#include "xdp/profile/database/database.h"
+
+namespace xdp {
+
+  AIEDebugWriter::AIEDebugWriter(const char* fileName,
```

Comment:
We found the following format to be a little easier to read and updated in the future:
```
  AIEDebugWriter::AIEDebugWriter(const char* fileName,
               const char* deviceName, uint64_t deviceIndex) 
    : VPWriter(fileName)
    , mDeviceName(deviceName)
    , mDeviceIndex(deviceIndex)
```